crates/weaver_forge/src/file_loader.rs

```diff
@@ -21,7 +21,28 @@ pub trait FileLoader {
     fn all_files(&self) -> Vec<PathBuf>;
 
     /// Returns the content of a file from a given name.
-    fn load_file(&self, file: &str) -> Result<Option<String>, Error>;
+    fn load_file(&self, file: &str) -> Result<Option<FileContent>, Error>;
+}
+
+/// A struct that represents the content of a file.
+#[derive(Debug)]
+pub struct FileContent {
```

Comment:
I'm not sure we want to preserve the full string contents of files we find before we parse the weaver config.  We should leave room for lazy loading, or Load + drop of the contents.

This is acceptable to fix post-this PR, but I don't think the struct as-is should be public and used across crates.

Reply:
Fixed in [582516b](https://github.com/open-telemetry/weaver/pull/231/commits/582516bf868d62e5c58c315eae33f283f00d46ee). I removed FileContent from the public API.